<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimedEvent</class>
 <widget class="QDialog" name="TimedEvent">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timed event</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QLineEdit" name="eventNameText">
       <property name="placeholderText">
        <string>Event name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="eventMessageText">
       <property name="placeholderText">
        <string>Type event message text</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="eventAffectsHuman">
         <property name="text">
          <string>affects human</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="eventAffectsCpu">
         <property name="text">
          <string>affects AI</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Day of first occurrence</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="eventFirstOccurrence">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Repeat after (0 = no repeat)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="eventRepeatAfter"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_18">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Affected players</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="playersAffected">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Resources</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="resources">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>16</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>16</number>
       </attribute>
       <column>
        <property name="text">
         <string>type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>qty</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
